core/src/main/java/ch/cyberduck/ui/browser/SearchFilter.java

```diff
@@ -72,8 +72,9 @@ public String toString() {
     public Pattern toPattern() {
         final StringBuilder pattern = new StringBuilder();
         pattern.append("\\A");
```

Comment:
Leading the regex with `(?i)` does the trick. No other lower case operations needed.

Reply:
Added to iterate-ch/docs#320.